mycore-base/src/main/java/org/mycore/common/MCRDefaultHTTPClient.java

```diff
@@ -0,0 +1,100 @@
+package org.mycore.common;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+
+import org.apache.http.client.cache.HttpCacheContext;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.cache.CacheConfig;
+import org.apache.http.impl.client.cache.CachingHttpClients;
+import org.apache.logging.log4j.LogManager;
+import org.mycore.common.config.annotation.MCRProperty;
+import org.mycore.common.content.MCRContent;
+import org.mycore.common.content.MCRStreamContent;
+import org.mycore.common.events.MCRShutdownHandler;
+import org.mycore.services.http.MCRHttpUtils;
+
+public class MCRDefaultHTTPClient implements MCRHTTPClient {
+    static org.apache.logging.log4j.Logger logger = LogManager.getLogger();
```

Comment:
when editing this class anyways: maybe use import statement instead of fully qualified name

Reply:
oh, of course, totally missed this...